src/components/DeviceAction/rendering.js

```diff
@@ -444,6 +444,54 @@ export function renderLoading({
   );
 }
 
+export function renderExchange({
+  exchangeType,
+  t,
+  device,
+  theme,
+}: RawProps & {
+  exchangeType: number,
+  device: Device,
+}) {
+  switch (exchangeType) {
+    case 0x00:
+      return <div>{"Confirm swap on your device"}</div>;
+    case 0x01:
+    case 0x02:
```

Comment:
We could add comments for each exchange type to improve clarity and documentation

```suggestion
    case 0x00: // swap
      return <div>{"Confirm swap on your device"}</div>;
    case 0x01: // sell
    case 0x02: // fund
```

Reply:
added